### PR TITLE
Fix python3

### PR DESCRIPTION
**Pull request by aaron-prindle**

### CODE REVIEW

File: ftl/common/args.py [hunk 1]
```diff
@@ -123,24 +123,51 @@ def base_parser():
...
                 'store',
                 "default":
                 "python2.7",
                 "help":
-                'The python version to use, formatted as pythonX.X \
+                'The python command to be run \
                 (ex: python2.7)'
             }
         ],
+        'pip_cmd': [
```
Comment (dlorenc) [inline ftl/common/args.py]: nit: I find these easier to read if you collapse the key: value to one line.

File: ftl/integration_tests/ftl_python_integration_tests_yaml.py [hunk 2]
```diff
@@ -7,8 +7,11 @@
 
 
 # Add directories for new tests here.
+# TEST_DIRS = [
```
Comment (dlorenc) [inline ftl/integration_tests/ftl_python_integration_tests_yaml.py]: Why is this commented out?

File: ftl/python/builder.py [hunk 3]
```diff
@@ -68,12 +70,13 @@ def Build(self):
 
             with ftl_util.Timing("resolving whl paths"):
                 whls = self._resolve_whls()
+                logging.info("WHLS:\n" + str(whls))
```
Comment (dlorenc) [inline ftl/python/builder.py]: You can use string formatting to make this cleaner.

File: ftl/python/builder.py [hunk 4]
```diff
@@ -132,9 +137,24 @@ def _whl_to_fslayer(self, whl):
...
+        pip_cmd_args.extend(['install', '--no-deps', '--prefix', pkg_dir, whl])
+
+        proc_pipe = subprocess.Popen(
+            pip_cmd_args,
+            stdin=subprocess.PIPE,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            env=self._gen_pip_env(),
+        )
+        stdoutdata, stderrdata = proc_pipe.communicate()
```
Comment (dlorenc) [inline ftl/python/builder.py]: just stdout and stdin?

File: out.txt [hunk 5]
```diff
@@ -0,0 +1,115 @@
+2018/02/24 17:32:13 Warning: The server docker version installed (17.09.0-ce) is different from the one used in GCB (17.06.1-ce)
```
Comment (dlorenc) [inline out.txt]: Remove this file?

File: ftl/integration_tests/ftl_python_integration_tests_yaml.py (hunk 2 above)
Comment (aaron-prindle) [reply ftl/integration_tests/ftl_python_integration_tests_yaml.py]: debugging, fixed now

File: ftl/python/builder.py (hunk 3 above)
Comment (aaron-prindle) [reply ftl/python/builder.py]: removing this statements as they were for debugging

File: ftl/python/builder.py (hunk 4 above)
Comment (aaron-prindle) [reply ftl/python/builder.py]: i have changed this to just stdout and stderror

File: ftl/common/args.py (hunk 1 above)
Comment (aaron-prindle) [reply ftl/common/args.py]: done

**Pull request merged**